### PR TITLE
Updates to readme for dev setup

### PR DESCRIPTION
**Pull request by vandercloak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,16 @@ The CSV files exported are designed to be used with [LosslessCut](https://github
...
+- First you will need a project setup in clerk. 
+  - Create a new project and grab your public + secret keys from the API Keys page.
+  - Copy them.
+  - Save them in a .env.local file.
+- Next you will need a twitch application (https://dev.twitch.tv/docs/authentication/)
+  - Paste them in to your .env.local under `TWITCH_CLIENT_SECRET` + `TWITCH_CLIENT_ID`
+- Install the node modules (make sure you are on node 16+)
+  (if you need to work with multiple versions of node then I highly recommend nvm!)
+- Start up the app with `npm run dev`
+- And presto. Should have running next app!
```
Comment (flbn) [inline README.md:33]:
```suggestion
### Create Your .env File
1) Set up the project with [Clerk](https://clerk.com/).
    - Create a new project
    - Copy your Public and Secret keys and save them in a `.env.local` file.
    - For troubleshooting and more information on how to setup Clerk, visit their documentation [here](https://clerk.com/docs).
2) Next, you will need a [Twitch](https://dev.twitch.tv/docs/authentication/) application
    - Copy your Twitch application ID and Secret Key into your `.env.local`
### Run Locally
1) Install dependencies 
    - `npm install`
2) Run in development
    - `npm run dev`
```

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+v16
```
Comment (flbn) [inline .nvmrc:1]: you may as well also add this to the `package.json` as the engine. on my phone rn so i can't really do anything other than suggest

File: README.md (hunk 1 above)
Comment (t3dotgg) [reply README.md:33]: I think that you have to use a custom Clerk OAuth credential for Twitch as well to get the `user:broadcast:read` permission

Comment (markflorkowski) [reply README.md:33]: confirmed: 
image

**Comment by t3dotgg** [comment]
Trumped https://github.com/pingdotgg/markerthing/pull/53

**Pull request closed without merging**